cove_360/tests.py

```diff
@@ -286,21 +287,21 @@
                   "on why these grants are £0 might be useful to anyone using the data, "
                   "so consider adding an explanation to the description of the grant.")},
      ['grants/0/amountAwarded'],
-     [['grants', 'Q', 2, 'Amount Awarded']]),
+     [{'sheet': 'grants', 'letter': 'Q', 'row_number': 2, 'header': 'Amount Awarded'}]),
```

Comment:
This has been changed from list of lists to list of dicts to be in agreement with the data structure for validation spreadsheet source path for schema validation